lib/src/viewport/states/camera_viewport_state.dart

```diff
@@ -0,0 +1,40 @@
+part of mapbox_maps_flutter;
+
+/// Manually sets camera to specified properties.
+final class CameraViewportState extends ViewportState {
```

Comment:
I get a crash anytime I try to use this (like in the simple example from the Changelog below). 

> Exception has occurred.
> ArgumentError (Invalid argument: Instance of 'CameraOptions')

```
MapWidget(
  viewport: CameraViewportState(
    center: Point(coordinates: Position(-117.918976, 33.812092)),
    zoom: 15.0,
  ),
);
```

Reply:
Whoops, thanks for catching this! I must have forgotten to push forward declarations for types in CameraViewportState, fixed!